api-server/src/server/boot/settings.js

```diff
@@ -115,14 +115,24 @@ function updateMyEmail(req, res, next) {
 // }
 
 function updateMyPortfolio(...args) {
-  const portfolioKeys = ['id', 'title', 'description', 'url', 'image'];
+  const portfolioKeys = [
+    'id',
+    'title',
+    'description',
+    'url',
+    'image',
+    'isSaved'
+  ];
   const buildUpdate = body => {
     const portfolio = body?.portfolio?.map(elem => _.pick(elem, portfolioKeys));
+    console.log(JSON.stringify(portfolio));
```

Comment:
```suggestion
```

Reply:
Indeed, ive done that. I ammended the last commit, so that change wouldnt need to be in its own commit. Hope that is ok.